### PR TITLE
Fixed some grammatical errors

### PR DESCRIPTION
**Pull request by Dishebh**
Things changed and errors found:

- Fixed: Added a comma to signify a brief pause wherever necessary.
- Fixed: Changed the singularity/plurality of words as per the sentence.
- Fixed: Removed unnecessary word(s) used in the sentence(s).

### CODE REVIEW

**Comment by Dishebh** [comment]
@jaredmorgs @shreyas1599 please review.

File: how-to/about-how-to.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Make sure you meet the following prerequisites before following the steps:
 
 * API credentials for the {company} v3.5 API.
 * Access to the Postman application.
-* An understanding about what a RESTful API is conceptually.
+* An understanding about what a RESTful API is, conceptually.
```
Comment (jaredmorgs) [inline how-to/about-how-to.md:43]: What do you think about swapping out the "about" with an "of":

An understanding of what a RESTful API is, conceptually.

File: tutorial/about-tutorial.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Make sure you meet the following prerequisites before starting the tutorial step
 
 * API credentials for the {company} v3.5 API.
 * Access to the Postman application.
-* An understanding about what a RESTful API is conceptually.
+* An understanding about what a RESTful API is, conceptually.
```
Comment (jaredmorgs) [inline tutorial/about-tutorial.md:51]: If you agree with the review comment above for swapping "about" with "of" then make the same change here.

File: tutorial/about-tutorial.md [hunk 3]
```diff
@@ -91,9 +91,9 @@ Conversational procedural structure, with a few ordered steps described in the b
 No outcome statement.
 ```
 
-You get a more conversational flow with this tutorial style, however some of the prescriptive clarity is lost by not having the headings.
+You get a more conversational flow with this tutorial style, however, some of the prescriptive clarity is lost by not having the headings.
```
Comment (jaredmorgs) [inline tutorial/about-tutorial.md:94]: Suggest splitting into two sentences:

You get a more conversational flow with this tutorial style. 
However, some of the prescriptive clarity is lost by not having the headings.

Comment (waldyrious) [inline tutorial/about-tutorial.md:94]: I'd actually convert the first comma to a semicolon:

```suggestion
You get a more conversational flow with this tutorial style; however, some of the prescriptive clarity is lost by not having the headings.
```

File: how-to/about-how-to.md (hunk 1 above)
Comment (Dishebh) [reply how-to/about-how-to.md:43]: Sure; seems good to me, too.

File: tutorial/about-tutorial.md (hunk 2 above)
Comment (Dishebh) [reply tutorial/about-tutorial.md:51]: Sure.

File: tutorial/about-tutorial.md (hunk 3 above)
Comment (Dishebh) [reply tutorial/about-tutorial.md:94]: Guess I'll go with the first suggestion.

Comment (Dishebh) [reply tutorial/about-tutorial.md:94]: Sure thing.

**Comment by Dishebh** [comment]
@jaredmorgs @waldyrious done with changes. Thank you!

File: tutorial/about-tutorial.md (hunk 3 above)
Comment (jaredmorgs) [reply tutorial/about-tutorial.md]: Yep the semicolon approach is fine here too.

File: tutorial/about-tutorial.md (hunk 2 above)
Comment (jaredmorgs) [reply tutorial/about-tutorial.md]: Will you push another change for this suggestion @Dishebh ?

File: how-to/about-how-to.md (hunk 1 above)
Comment (jaredmorgs) [reply how-to/about-how-to.md]: Will you make this change and push another revision @Dishebh ?

Comment (Dishebh) [reply how-to/about-how-to.md]: I already committed the change. Please check out my commit with the commit message of "minor update of swapping out 'about' with an 'of'".

File: tutorial/about-tutorial.md (hunk 2 above)
Comment (Dishebh) [reply tutorial/about-tutorial.md]: Same here. Already committed the change.

**Comment by Dishebh** [comment]
@jaredmorgs I accidentally created this PR out of my `master` branch instead of creating and switching to a new branch. And now, I am facing some difficulty in checking out a different branch and making it in sync with the master branch of the original repo. If the changes seem fine, could you please approve and merge it? (It would save me a lot of time figuring out a workaround, otherwise).
Thank you so much!

**Comment by jaredmorgs** [comment]
Sorry @Dishebh the github diff comparison wasn't showing me your changes where I expected to see them. 😁 This is all looking good.

**Comment by Dishebh** [comment]
> Sorry @Dishebh the github diff comparison wasn't showing me your changes where I expected to see them.  This is all looking good.

Thank you so much for this :)

**Pull request merged**